ORM/NullableEmbeddableListener.php

```diff
@@ -16,9 +16,10 @@
     private $propertyMap = [];
     private $useClosure = false;
 
-    public function __construct(PropertyAccessorInterface $propertyAccessor)
+    public function __construct(PropertyAccessorInterface $propertyAccessor, bool $useClosure = false)
```

Comment:
I don't like optional constructor arguments. What's the benefit over the setter?